msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md

```diff
@@ -0,0 +1,102 @@
+keywords: camera image microphone capabilities native device permissions media qr barcode scanner
+ms.topic: conceptual
+ms.author: lajanuar
+---
+
+# Integrate QR or barcode scanner capability 
+
+This document guides you on how to integrate QR or barcode scanner capability of the mobile phone with Teams platform. 
+You can read the code with an optical scanner feature of the native device camera. For richer collaborative experience, you can combine the scanner capability with the Teams platform through the integration process. 
+
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), that provides the tools necessary for your app to access the user’s [device permissions](native-device-permissions.md). After getting the camera access of the device, use **scanBarCode API** to integrate the scanner capability with Teams within your Microsoft Teams mobile app, and build a richer experience. 
+
+## Advantage of integrating QR or barcode scanner capability
+
+The main advantage of integrating QR or barcode scanner capability in your Teams apps is to help the developers on Teams platform to leverage QR or barcode scanning functionality with Teams JavaScript Client SDK.
+
+To integrate QR or barcode scanner capability, you must update the app manifest file and call the **`scanBarCode` API**. For effective integration, you must have a good understanding of [code snippets](#code-snippets) for calling the `scanBarCode` API, which allows you to use native QR or barcode scanner capability. The API throws error in case of an unsupported barcode standard.
+It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your Teams app.
+
+> [!NOTE] 
+> Currently, Microsoft Teams support for QR or barcode scanner capability is only available for mobile clients.
+
+## Update manifest
+
+Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#devicepermissions) file by adding the `devicePermissions` property and specifying `media`. It allows your app to ask for requisite permissions from users before they start using  the QR or barcode scanner for integration.
+
+``` json
+"devicePermissions": [
+    "media",
+],
+```
+
+> [!NOTE]
+> The **Request Permissions** prompt is automatically displayed when a relevant Teams API is initiated. For more information, see [Request device permissions](native-device-permissions.md).
+
+## ScanBarCode API
+
+The `ScanBarCode` API invokes scanner control that enables the user to scan different types of QR or barcode, and returns the scanned result as a string.
+Following are the three main steps to use QR or barcode scanner capability API:
+
+1. Entry point or invoke: 
+The entry point is the point from where a developer invokes `ScanBarCode` API. This point is a button with the label **Scan barcode**. 
```

Comment:
Self-review check: This doc is intended for developers, shouldn't it be "you" and not developers?